crates/bevy_ui/src/render/mod.rs

```diff
@@ -232,6 +237,23 @@ pub fn extract_uinode_background_colors(
             // so we have to divide by `UiScale` to get the size of the UI viewport.
             / ui_scale.0;
 
+        // Both vertical and horizontal percentage border values are calculated based on the width of the parent node
+        // <https://developer.mozilla.org/en-US/docs/Web/CSS/border-width>
```

Comment:
layout and style is a complex topic, having these comments that refers back to source material helps a lot, thanks!